src/index.html

```diff
@@ -17,16 +17,54 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="top__bar"
+      method="post"
+    >
+      <div class="search__box">
+        <a
+          href="#"
+          class="search__glass"
+        >
+          <img
+            src="images/Search.svg"
+            alt="search image"
+            class="search__img-big"
+          />
+        </a>
```

Comment:
Redundant, because you need to add icon with background property 
<img width="545" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/7a9594c0-5069-498d-bf57-b1489985c43a">
```suggestion
```